spinalcordtoolbox/reports/qc2.py

```diff
@@ -267,19 +268,22 @@ def sct_deepseg(
         path_qc=Path(path_qc),
         command=command,
         cmdline=list2cmdline(cmdline),
-        plane='Axial',
+        plane=plane,
         dataset=dataset,
         subject=subject,
     ) as imgs_to_generate:
         if "seg_spinal_rootlets_t2w" in argv:
             sct_deepseg_spinal_rootlets_t2w(
                 imgs_to_generate, fname_input, fname_seg, fname_seg2, species)
-        else:
-            sct_deepseg_default(
+        if plane == 'Axial':
+            sct_deepseg_axial(
+                imgs_to_generate, fname_input, fname_seg, fname_seg2, species)
+        elif plane == 'Sagittal':
```

Comment:
Minor nitpick, but by using `if/elif` with no `else`, there's theoretically a chance that this could be called with a `plane` value other than Axial/Sagittal, and then the behavior would be unclear.

So, it tends to be safer to use the following pattern (even if you're 99% sure that you'll only ever get Axial/Sagittal). Like, even though -qc-plane only has 2 options, maybe in the future the `sct_deepseg()` QC function gets used somewhere else, and someone makes a typo... who knows! :stuck_out_tongue_closed_eyes:

```suggestion
        else:
            assert plane == 'Sagittal', (f"`plane` must be either 'Axial' "
                                         f"or 'Sagittal', but got {plane}")
```

That way, all of the non-Axial cases will go into the "else", and we'll do an internal check to make sure that `plane` is supposed to be 'Sagittal'.

Reply:
Ah, I don't want to hold this PR up... It's really not a big deal.